example/spec/syncGranule/SyncGranuleSuccessSpec.js

```diff
@@ -0,0 +1,72 @@
+const fs = require('fs');
+const path = require('path');
+const { s3, s3ObjectExists } = require('@cumulus/common/aws');
+const { buildAndExecuteWorkflow, LambdaStep } = require('@cumulus/integration-tests');
+
+const { loadConfig, templateFile } = require('../helpers/testUtils');
+const config = loadConfig();
+const lambdaStep = new LambdaStep();
+const taskName = 'SyncGranule';
+
+const outputPayloadTemplateFilename = './spec/syncGranule/SyncGranule.output.payload.template.json'; // eslint-disable-line max-len
+const templatedOutputPayloadFilename = templateFile({
+  inputTemplateFilename: outputPayloadTemplateFilename,
+  config: config.IngestGranule.SyncGranuleOutput
+});
+const expectedPayload = JSON.parse(fs.readFileSync(templatedOutputPayloadFilename));
+
+
+describe('The Sync Granules workflow', () => {
+  const inputPayloadFilename = './spec/syncGranule/SyncGranule.input.payload.json';
+  const inputPayload = JSON.parse(fs.readFileSync(inputPayloadFilename));
+  const collection = { name: 'MOD09GQ', version: '006' };
+  const provider = { id: 's3_provider' };
+  let workflowExecution = null;
+
+  beforeAll(async () => {
+    // eslint-disable-next-line function-paren-newline
+    workflowExecution = await buildAndExecuteWorkflow(
+      config.stackName, config.bucket, taskName, collection, provider, inputPayload
+    );
+  });
+
+  it('executes successfully', () => {
```

Comment:
Change this to 'completes execution with success status' (again addressing Patrick's comment in the chat - we'll have to start changing these)